packages/eas-cli/src/credentials/ios/actions/AppleAppIdentifierUtils.ts

```diff
@@ -0,0 +1,4 @@
+export function isWildcardBundleIdentifier(bundleIdentifier: string): boolean {
+  const wildcardRegex = /^[A-Za-z0-9.-]+\*$/;
```

Comment:
maybe let's keep regexps like this in one place e.g. packages/eas-cli/src/project/ios/bundleIdentifier.ts